frontend/src/pages/SendMetadata/MetadataForm.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux';
+import { Redirect, Link } from 'react-router-dom';
+
+import { alertActions } from '../../state/actions/alert';
+
+const Wrapper = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+`;
+
+const StyledForm = styled.form`
+  flex: 1;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-content: space-around;
+`;
+
+const Input = styled.input`
+  flex: 0 0 2em;
+  margin: 0.5em 1em;
+`;
+
+const Select = styled.select`
+  flex: 0 0 2em;
+  margin: 0.5em 1em;
+`;
+
+const TextArea = styled.textarea`
+  flex: 0 0 10em;
+  margin: 0.5em 1em;
+`;
+
+const RadioLabel = styled.label`
+  border-radius: 0.2em;
+  padding: 0.2em;
+  ${({ background }) => (background ? `background-color: ${background};` : '')}
+  ${({ border }) => (border ? `border: 1px solid ${border};` : '')}
+`;
+
+const HorizontalWrapper = styled.div`
+  flex: 0;
+  display: flex;
+  flex-direction: row;
+`;
+
+export const MetadataForm = () => {
+  const [state, setState] = useState({
+    metadataTypeName: '',
+    releaseState: 1,
+    description: '',
+    formatName: '',
+    municipalityName: '',
+    url: '',
+  });
+
+  // municipalities should be objects with a "name" key
+  const [municipalities, setMunicipalities] = useState([]);
+  const [metadataTypes, setMetadataTypes] = useState([]);
+  const dataFormats = ['JSON', 'CSV'];
+
+  const [submissionStatus, setSubmissionStatus] = useState('');
+
+  const dispatch = useDispatch();
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+    const nextState = { ...state };
+    nextState[name] = value;
+    setState(nextState);
+  };
+
+  const handleRadioChange = (event) => {
+    const { name, value } = event.currentTarget;
+    const nextState = { ...state };
+    nextState[name] = parseInt(value, 10);
+    setState(nextState);
+  };
+
+  // fetch municipalities
+  useEffect(() => {
```

Comment:
This is not really necessary, when the backend support multiple municipalities it should just be fetched from the user object in redux.